### PR TITLE
release: version `v10.5.0`

### PR DESCRIPTION
**Pull request by georglauterbach**
# Description

This PR marks the feature freeze phase for the `v10.5.0` release.

The current `:edge` image will be tested for roughly one week and if no bugs get reported, `v10.5.0` will be released. During this testing period, only PRs that do not change functionality will be merged.

**If you want to support us, give the latest `:edge` image a try and report any issues you encounter back to this PR.**

Documentation updates may be merged. **Feature freeze is in place until _01 March 2022_**. 

(Previously) Blocking:

- [x] #2424
- [x] #2428

Until it is decided whether they go into this release or not, `10.5.0`'s feature free is not in place.

EDIT: I seems as if a decision was reached. #2424 and #2428 are breaking changes and require a major version bump.

[//]: # (until _24 Dec 2021_)

## Type of change

- [x] Release

## Release-Checklist:

- [x] Update CHANGELOG.md
- [x] Update version in VERSION
- [x] Draft release

### CODE REVIEW

File: CHANGELOG.md
Comment (wernerfred) [inline CHANGELOG.md:11]:
```suggestion
### Other Minor Changes
```

**Review by wernerfred — CHANGES_REQUESTED** [review]
Thanks for Putting together all PRs :rocket:
For Future us: We could think of Auto-generating the Changelog / Release content through CI

File: CHANGELOG.md
Comment (wernerfred) [inline CHANGELOG.md:16]: Should we add the corresponding PRs here too?

Comment (georglauterbach) [reply CHANGELOG.md]: solved in [21d6b2c](https://github.com/docker-mailserver/docker-mailserver/pull/2431/commits/21d6b2c8fe9f8dff77b666ce434a360a938f8bc6)

File: CHANGELOG.md
Comment (georglauterbach) [reply CHANGELOG.md]: solved in [21d6b2c](https://github.com/docker-mailserver/docker-mailserver/pull/2431/commits/21d6b2c8fe9f8dff77b666ce434a360a938f8bc6)

**Review by casperklein — APPROVED** [review]
> If you want to support us, give the latest :edge image a try and report any issues you encounter back to this PR.

No problems in my setup 👍 I thought about, how to get more people testing the edge image. There seems to be no way to pin PRs like issues. Maybe next time, we open/pin also a an issue about the upcoming release?

**Comment by georglauterbach** [comment]
> I thought about, how to get more people testing the edge image. There seems to be no way to pin PRs like issues. Maybe next time, we open/pin also a an issue about the upcoming release?

We could, as an alternative, open a discussion (type: announcement). Maybe this is more fitting.

**Comment by georglauterbach** [comment]
What say ye @docker-mailserver/maintainers? :D

**Review by NorseGaud — APPROVED** [review]

**Comment by NorseGaud** [comment]
I see nothing that would cause me to reject it :)

**Comment by georglauterbach** [comment]
Release was scheduled for today, but this PR lacks approval. Currently, changes requested by @wernerfred are blocking and @polarathene has not approved either. There is no rush, just wanted to let you know.

**Review by wernerfred — COMMENTED** [review]
Sorry. I didn't approve in a first place as I hadn't time to test `edge`. Didn't notice that the change request is still blocking after it got adressed. Hope this "Reviewal Comment" will superseed the Change Request.

EDIT: Seems like a comment will only be added to the reviews, not replace a previous one. I'll find a way to remove the blocker

**Comment by polarathene** [comment]
> @polarathene has not approved either

I still don't actually run this project, only contribute/maintain :sweat_smile: 

I can approve but I've not done any testing on my end.

**Review by polarathene — APPROVED** [review]

**Comment by georglauterbach** [comment]
> > @polarathene has not approved either
> 
> I still don't actually run this project, only contribute/maintain sweat_smile
> 
> I can approve but I've not done any testing on my end.

While this is true, there is no hard requirement that forces you to use `:edge` to approve this PR. Otherwise I assume there is a very valid reason for _not_ approving this PR so I will not merge it :)

**Pull request merged**
